vscode-extension/package.json

```diff
@@ -5,14 +5,43 @@
   "author": "LastMile AI",
   "displayName": "AIConfig Editor",
   "description": "AIConfig notebook editor that turns VSCode into a generative AI studio.",
-  "version": "0.0.18",
+  "version": "0.0.20",
   "license": "MIT",
   "engines": {
     "vscode": "^1.85.0"
   },
   "icon": "static/Logo.png",
   "categories": [
-    "Other"
+    "Machine Learning",
+    "Notebooks",
+    "Programming Languages",
+    "Snippets"
+  ],
+  "keywords": [
+    "ai",
+    "assistant",
+    "chatbot",
+    "notebook",
+    "aiconfig",
+    "lastmile",
+    "openai",
+    "chatgpt",
+    "gpt",
+    "gpt4",
+    "dalle",
+    "dalle3",
+    "llama",
+    "llama2",
+    "anthropic",
+    "claude",
+    "claude3",
+    "gemini",
+    "huggingface",
+    "transformers",
+    "generative",
+    "editor",
+    "python",
+    "typescript"
```

Comment:
just a headsup - the vscode extension marketplace website shows all the tags

<img width="1564" alt="Screenshot 2024-03-11 at 5 23 45 PM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/9bc55894-0491-49d7-9bea-c13dc0142dee">

Reply:
Good question. Will defer this to @andrew-lastmile.